.github/workflows/tests.yml

```
@@ -1,112 +1,238 @@
-name: Build Only (No Tests)
+name: Tests
 
 on:
   push:
-    branches: [ main, develop ]
+    branches: [ main, develop, category-normalization ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)